ivy/functional/ivy/activations.py

```diff
@@ -620,8 +628,13 @@ def mish(
 @handle_out_argument
 @to_native_arrays_and_back
 @handle_array_function
+@handle_complex_input
 def hardswish(
```

Comment:
`hardswish` is missing a `jax_like` function. Complex numbers are marked as unsupported in the `tf` and `torch` backends.

Reply:
Done